.golangci.yml

```diff
@@ -73,6 +73,25 @@ linters-settings:
           - github.com/stretchr/testify/require
           - github.com/syndtr/goleveldb
           - github.com/decred/dcrd/dcrec/secp256k1/v4
+          - google.golang.org/protobuf/proto
```

Comment:
Why are we changing the linter here?

Reply:
One of the commits in the original PR was a lint fix. When I ran `make lint` on this branch, it complained that the `depguard` linter doesn't allow some dependencies. This did not show in the v1 work because `make lint` runs `pre-commit` on v1, while in v0.38, it runs `golangci-lint` directly, and they seem to have different runtime parameters. If you run golangci-lint directly on v1, you can see the same depguard issues.

I believe that for v0.38, it's best to fix the depguard list. In v1, goangci-lint is not run as strictly.